src/partials/mobile-tablet-menu.html

```diff
@@ -0,0 +1,35 @@
+<div class="menu-overlay">
+  <nav class="menu">
+    <div class="menu-close">
+      <svg viewBox="0 0 40 40">
+        <use href="./img/sprite.svg#icon-close"></use>
+      </svg>
+    </div>
```

Comment:
please use <button>  for this solution.

Reply:
didn't get what to use for this solution :)

Reply:
fixed